.github/workflows/build-wheels.yml

```diff
@@ -58,17 +71,19 @@ jobs:
           # different, and are containerised fully, therefore they get
           # their toolchain path from pyproject.toml
           CMAKE_TOOLCHAIN_FILE: ${{ github.workspace }}/.conan/conan_paths.cmake
-          # We matrix the python build here ourselves, rather than
-          # letting cibuildwheel do its regular python matrix, as we
-          # want to allow a full release pipeline to run "vertically" on
-          # CI, on individual agents, rather than building every python
-          # version sequentially on each platform. This is done less for
-          # performance reasons (although it does help) and more so a
-          # single failing python version won't interrupt every other
-          # deploy on that platform.
-          CIBW_BUILD: ${{ matrix.python-build }}
           CIBW_SKIP: '*musllinux* *arm64*'
-          CIBW_MANYLINUX_X86_64_IMAGE: manylinux2014
+          # Python 3.9 implies VFX Reference Platform CY22, which
+          # specifies glibc 2.17 and the deprecated libstdc++ ABI, and
+          # is typically associated with CentOS 7. The manylinux2014
```

Comment:
CentOS 7 and older. e.g. Ubuntu was once using that ABI too.

What I'm calling out is that it's not specific to CentOS version 7.

Reply:
Yeah I was careful to say "typically associated with" since that's predominant in VFX - it feeds into the next point about the manylinux2014 container being based on CentOS 7.

Reply:
Fair point on the wording.
An EL7 derivative is what most VFX users would be aware of, and commonly use, so I'll resolve this.